specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
I removed `ballot_manifest_info` in my recent commit (9994328).  In this spec, I am, in fact, using ballot to represent the encoding of the auditors, so it is wise for us to add a new type (aCVR is fine with me) to represent that information.  Go ahead and add timestamps to the model too.  Right now logging is completely underspecified because we do not really know what we must log (my contract and law) and what we should log (for forensic auditing and debugging).